test/ConfigLoader.test.ts

```diff
@@ -329,193 +319,162 @@ describe('ConfigLoader', () => {
         path: 'proxy.config.json',
         branch: 'main',
         enabled: true,
-      };
+      } as GitSource;
 
       const config = await configLoader.loadFromSource(source);
 
       // Verify the loaded config has expected structure
-      expect(config).to.be.an('object');
-      expect(config).to.have.property('cookieSecret');
-    });
+      expect(config).toBeTypeOf('object');
+      expect(config).toHaveProperty('cookieSecret');
+    }, 10000);
 
-    it('should throw error for invalid configuration file path (git)', async function () {
+    it('should throw error for invalid configuration file path (git)', async () => {
       const source = {
         type: 'git',
         repository: 'https://github.com/finos/git-proxy.git',
         path: '\0', // Invalid path
         branch: 'main',
         enabled: true,
-      };
+      } as GitSource;
 
-      try {
-        await configLoader.loadFromSource(source);
-        throw new Error('Expected error was not thrown');
-      } catch (error) {
-        expect(error.message).to.equal('Invalid configuration file path in repository');
-      }
+      await expect(configLoader.loadFromSource(source)).rejects.toThrow(
+        'Invalid configuration file path in repository',
+      );
     });
 
-    it('should throw error for invalid configuration file path (file)', async function () {
+    it('should throw error for invalid configuration file path (file)', async () => {
       const source = {
         type: 'file',
         path: '\0', // Invalid path
         enabled: true,
-      };
+      } as FileSource;
 
-      try {
-        await configLoader.loadFromSource(source);
-        throw new Error('Expected error was not thrown');
-      } catch (error) {
-        expect(error.message).to.equal('Invalid configuration file path');
-      }
+      await expect(configLoader.loadFromSource(source)).rejects.toThrow(
+        'Invalid configuration file path',
+      );
     });
 
     it('should load configuration from http', async function () {
-      this.timeout(10000);
-
       const source = {
         type: 'http',
         url: 'https://raw.githubusercontent.com/finos/git-proxy/refs/heads/main/proxy.config.json',
         enabled: true,
-      };
+      } as HttpSource;
 
       const config = await configLoader.loadFromSource(source);
 
       // Verify the loaded config has expected structure
-      expect(config).to.be.an('object');
-      expect(config).to.have.property('cookieSecret');
-    });
+      expect(config).toBeTypeOf('object');
+      expect(config).toHaveProperty('cookieSecret');
+    }, 10000);
 
-    it('should throw error if repository is invalid', async function () {
+    it('should throw error if repository is invalid', async () => {
       const source = {
         type: 'git',
         repository: 'invalid-repository',
         path: 'proxy.config.json',
         branch: 'main',
         enabled: true,
-      };
+      } as GitSource;
```

Comment:
Again, why the override? If its needed worth a comment. If its a shortcut, does it make the test more fragile?